### PR TITLE
Shrubbery

### PR DESCRIPTION
**Pull request by jtkorhonen**
O Knights of Ni, we have brought you your shrubbery. May we go now?

### CODE REVIEW

**Comment by jtkorhonen** [comment]
There’s no more work. We’re destitute. I’m afraid I have no choice but to sell you all for scientific experiments.

File: gizmo.py [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+    """Spells out the given `word`.
+
+    """
+    for i,letter in enumerate(word):
+        if i > 0: 
+            print('.', end='')
+        print(letter, end='')
+    print()
+
+    #print(".".join(word))   # <-- this is the correct pythonic solution
```
Comment (wmvanvliet) [inline gizmo.py:28]: True, but this was about messing around with loops.

File: gizmo.py [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+def get_titanic_children(titanic_csv: str="titanic.csv"):
+    """Reads and returns all the children from the `titanic_csv` file.
+
+    Returns
+    -------
+    pandas.DataFrame
+        Filtered out data of children on the Titanic.
+
+    """
+    return get_titanic(titanic_csv).query('age <= 12.0')
```
Comment (wmvanvliet) [inline gizmo.py:154]: You used `query`! Nice.

**Comment by wmvanvliet** [comment]
Nice work!

**Pull request closed without merging**